shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1357,16 +1357,15 @@ void WebContents::RenderViewCreated(content::RenderViewHost* render_view_host) {
   if (!background_throttling_)
     render_view_host->SetSchedulerThrottling(false);
 
-  // Set the background color of RenderWidgetHostView.
+  // If the RenderWidgetHostView contains a background color (is not
+  // transparent), set the background color earlier.
   auto* const view = web_contents()->GetRenderWidgetHostView();
   auto* web_preferences = WebContentsPreferences::From(web_contents());
   if (view && web_preferences) {
     std::string color_name;
     if (web_preferences->GetPreference(options::kBackgroundColor,
                                        &color_name)) {
       view->SetBackgroundColor(ParseHexColor(color_name));
-    } else {
-      view->SetBackgroundColor(SK_ColorTRANSPARENT);
```

Comment:
I'm not sure I understand why we don't want to set the background color to transparent here any more..?

Reply:
That's fair; I removed it just because it doesn't seem to be respecting the transparent option at all, and is setting the background color to white regardless, so I thought keeping it would be confusing for future debugging. Happy to leave it in though!

Reply:
I think this is required for the `transparent: true` option to work correctly